package/python-libroboint/python-libroboint.mk

```diff
@@ -4,14 +4,10 @@
 #
 ################################################################################
 
-PYTHON_LIBROBOINT_VERSION = b4dda4e5bad4f828096f44904c98f597b7c13be3
-PYTHON_LIBROBOINT_SITE = $(call github,PeterDHabermehl,libroboint-py3,$(PYTHON_LIBROBOINT_VERSION))
-PYTHON_LIBROBOINT_DEPENDENCIES = libroboint
-
-define PYTHON_LIBROBOINT_INSTALL_PY_STUFF
-	$(INSTALL) -D -m 0644 $(@D)/robointerface.py $(TARGET_DIR)/usr/lib/python$(PYTHON3_VERSION_MAJOR)/site-packages/robointerface.py
-endef
-PYTHON_LIBROBOINT_POST_INSTALL_TARGET_HOOKS += PYTHON_LIBROBOINT_INSTALL_PY_STUFF
-
-$(eval $(generic-package))
+PYTHON_LIBROBOINT_VERSION = cf25fdba52ad1343317afca683d8b272682f0077
+PYTHON_LIBROBOINT_SITE = $(call gitlab,Humpelstilzchen,libroboint,$(LIBROBOINT_VERSION))
```

Comment:
@ski7777 Is that correct?

Reply:
yes
We use the upstream package now. python code is in python subdir. nothing too special